kv/src/kv.ts

```diff
@@ -555,9 +555,9 @@ export class Bucket implements KV, KvRemove {
     return new KvStoredEntryImpl(this.bucket, this.prefixLen, sm);
   }
 
-  jmToEntry(jm: JsMsg): KvEntry {
+  jmToWatchEntry(jm: JsMsg, isUpdate = false): KvWatchEntry {
```

Comment:
Seems like `jmToWatchEntry` is always called with a value provided for `isUpdate`, could potentially remove the default.

Reply:
fixed!